docs/modules/tutorials/pages/tech-basics-for-newbies.adoc

```diff
@@ -0,0 +1,101 @@
+= Trainings
+:noindex:
+== VIVIDUS usage related technical basics
+
+Depending on which plugins/steps you are planning to use, it may be useful to complete the following technical basics trainings before processing to VIVIDUS:
+
+[cols="1,2,2", options="header"]
+|===
+
+|Topic
+|Goal
+|Resources
+
+|Git
+
+a|Get basic understand of managing the version control process with Git:
+
+    * repo cloning
+    * branching strategy, create/update/delete branches
+    * create/update/delete/push/merge/squash a commit, etc.
+    * merge conflicts resolution
+
+a| * https://git-scm.com/about
```

Comment:
it would be good to give names to all links (like it's done for `XPath validator` or `JsonPath evaluator`) so that users can understand what knowledge will link provide